doc/developer/debugging.rst

```diff
@@ -5,14 +5,31 @@ Debugging
 
 The goal of this document is to give some troubleshooting tips.
 
-General
+Webpack
 -------
 
-First you should add ``?debug`` in the application URL to have all the
-JavaScript and stylesheets in separated non-minified files.
+To have all the JavaScript and stylesheets in separated non-minified files.
```

Comment:
I do not understand the connection between the header "Webpack", this sentence ("To have all the JavaScript...") and the following instructions.
Is there something missing?

Reply:
It's Webpack who minify the js and the css, and we use webpack-dev-derver to don't have this minification...

Reply:
So we could say something like:
"For debugging purposes, it is better to have all the JavaScript and stylesheets in separated, non-minified files. To achieve this, you need to disable Webpack's minification; you can achieve this as follows.

What do you think?

Reply:
I just updated with a variant of your proposal :-)